docker/dev/Dockerfile

```diff
@@ -2,8 +2,7 @@ FROM node:12.13-alpine As development
 
 WORKDIR /app
 
-RUN apk add --no-cache bash curl openssl python make g++ && \
-  curl -sfL https://install.goreleaser.com/github.com/tj/node-prune.sh | sh && \
+RUN apk add --no-cache bash curl openssl python make g++ && npm prune && \
```

Comment:
I had to include this because this package `https://install.goreleaser.com/github.com/tj/node-prune.sh` does not support linux arm64 devices which is the architecture of apple m1.  using npm prune which basically does the same thing fixes this.